tests/Fixer/Import/NoUnusedImportsFixerTest.php

```diff
@@ -683,4 +683,47 @@ public function testWithSameNamespaceImportAndUnusedImportAfterNamespaceStatemen
 
         $this->doTest($expected, $input);
     }
+
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideFix72Cases
+     * @requires PHP 7.2
+     */
+    public function testFix72($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFix72Cases()
+    {
+        return array(
+            array( // TODO test shows lot of cases where imports are not removed while could be
```

Comment:
somehow we didn't spot it.
@SpacePossum could you take a look at it ?

Reply:
This is done on purpose; so we know this case (and other cases like group import) are not fixed by the fixer while it could.
I still left the case in to prove what happens on PHP7.2, i.e. the fixer doesn't crash nor makes incorrect changes. This way we prevent regression.
However, when someone wants to patch up the fixer to support this still should fail and the TODO message will tell why.
I patched up the other fixers, but this one was kind of to big to fix by me ATM.

Reply:
OK, good enough.
I notice the `todo` during merging 2.2->up, I didn't know the background here, that's why asking.